idanalysis/eqparams.py

```diff
@@ -661,10 +661,10 @@ def calc_i3(self):
         """
         by = self.field_profile[:, 1]
         bx = self.field_profile[:, 2]
-        x = 1e-3 * self.field_profile[:, 0]
-        dx = _np.diff(x)[0]
-        i3x = _np.trapz(dx=dx, y=_np.abs((by / self.brho) ** 3))
-        i3y = _np.trapz(dx=dx, y=_np.abs((bx / self.brho) ** 3))
+        s = 1e-3 * self.field_profile[:, 0]
+        ds = _np.diff(s)[0]
```

Comment:
Ff you only need the first element, you don't need to take the difference of the whole vector, right?
You could only do `ds = s[1] - s[0]`.